cmd/controller-manager/main.go

```diff
@@ -64,6 +66,8 @@ func main() {
 	flag.StringVar(&fedNamespace, "federation-namespace", util.DefaultFederationSystemNamespace, "The namespace the federation control plane is deployed in.")
 	flag.StringVar(&clusterNamespace, "registry-namespace", util.MulticlusterPublicNamespace, "The cluster registry namespace.")
 	flag.BoolVar(&limitedScope, "limited-scope", false, "Whether the federation namespace will be the only target for federation.")
+	flag.IntVar(&clusterAvailableDelay, "cluster-available-delay", util.DefaultClusterAvailableDelay, "Time in seconds to wait before reconciling on a healthy cluster.")
```

Comment:
Please use `flag.DurationVar`

Reply:
@tghartland, Please fix this and i think its ready to be merged.